pkgs/development/libraries/jemalloc/default.nix

```diff
@@ -19,6 +20,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-LbgtHnEZ3z5xt2QCGbbf6EeJvAU3mDw7esT3GJrs/qo=";
   };
 
+  patches = [
+    # fix tests under --with-jemalloc-prefix=, see https://github.com/jemalloc/jemalloc/pull/2340
+    (fetchpatch {
+      url = "https://github.com/jemalloc/jemalloc/commit/d00ecee6a8dfa90afcb1bbc0858985c17bef6559.patch";
+      sha256 = "sha256:11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p";
```

Comment:
is it ok to use `fetchpatch` with pull-request commits?
also not sure why but sha256 on my machine was not `1kq6ji7xq65a8szb5rzavi732r1j8izlvrvkhxhbg4hb4c7hp7ra` from suggestion but rather `11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p`

Reply:
`fetchpatch` does change the file, yes.

@divanorama How did you get `11wp6byrhxi8ymqgknb40xx5xcwyj60ydajqh2029qc924ivi61p`, did you just take the hash of the file as downloaded from GitHub?

Reply:
The hash in this PR is correct. The one I was suggesting was incorrect, since I just used nix-prefetch-url to obtain it, which is wrong (since fetchpatch does change the file).

Reply:
Obtained the silly way - tried to build, then copied from the error message. Same trick for when adding files - put random hash in, update from error message, is there a simple i_dont_know_the_hash_yet placeholder though?

Reply:
> is there a simple i_dont_know_the_hash_yet placeholder though?

`lib.fakeSha256`